examples/crm/src/contacts/ContactAside.tsx

```diff
@@ -66,33 +66,31 @@ export const ContactAside = ({ link = 'edit' }: { link?: 'edit' | 'show' }) => {
                     />
                 </Stack>
             )}
-            {record.phone_number1 && (
+            {record.phone_number1.number && (
```

Comment:
**question:** Will this be OK when we will use supabase to have a nested field ?

Reply:
Supabase normally supports unstructured data. So I don't think there will be any problems. At worst, we'll change it then